support/support.rotation

```diff
@@ -2,48 +2,24 @@
 #
 # Begin metadata
 #@ title: Serving
-#@ slack: #serving-questions
-#@ slacklink: https://cloud-native.slack.com/archives/C04LMU0AX60
-
-2021-09-27T01:00:00Z | dprotaso
-2021-10-04T01:00:00Z | carlisia
-2021-10-11T01:00:00Z | markusthoemmes
-2021-10-18T01:00:00Z | julz
-2021-10-25T01:00:00Z | ZhiminXiang
-2021-11-01T01:00:00Z | nak3
-2021-11-08T01:00:00Z | shinigambit
-2021-11-15T01:00:00Z | yanweiguo
-2021-11-22T01:00:00Z | nader-ziada
-2021-11-29T01:00:00Z | carlisia
-2021-12-06T01:00:00Z | dprotaso
-2021-12-13T01:00:00Z | julz
-2021-12-20T01:00:00Z | N/A - break
-2021-12-27T01:00:00Z | N/A - break
-2022-01-03T01:00:00Z | TBD
-2022-01-10T01:00:00Z | nader-ziada
-2022-01-17T01:00:00Z | nak3
-2022-01-24T01:00:00Z | shinigambit
-2022-01-31T01:00:00Z | yanweiguo
-2022-02-07T01:00:00Z | ZhiminXiang
-2022-08-01T01:00:00Z | carlisia
-2022-08-08T01:00:00Z | dprotaso
-2022-08-15T01:00:00Z | nader-ziada
-2022-08-22T01:00:00Z | nak3
-2022-08-29T01:00:00Z | psschwei
-2022-09-05T01:00:00Z | carlisia
-2022-09-12T01:00:00Z | dprotaso
-2022-09-19T01:00:00Z | nader-ziada
-2022-09-26T01:00:00Z | nak3
-2022-10-03T01:00:00Z | psschwei
-2022-10-10T01:00:00Z | carlisia
-2022-10-17T01:00:00Z | dprotaso
-2022-10-24T01:00:00Z | nader-ziada
-2022-10-31T01:00:00Z | nak3
-2022-11-07T01:00:00Z | psschwei
-2022-11-14T01:00:00Z | carlisia
-2022-11-21T01:00:00Z | dprotaso
-2022-11-28T01:00:00Z | nader-ziada
-2022-12-05T01:00:00Z | nak3
-2022-12-12T01:00:00Z | psschwei
-2022-12-19T01:00:00Z | N/A - break
-2022-12-26T01:00:00Z | N/A - break
+#@ slack: #knative, #knative-serving
+#@ slacklink: https://cloud-native.slack.com/archives/C04LGHDR9K7, https://cloud-native.slack.com/archives/C04LMU0AX60
+#
+2023-02-27T01:00:00Z | dprotaso
+2023-03-06T01:00:00Z | KauzClay
+2023-03-13T01:00:00Z | ReToCode
+2023-03-20T01:00:00Z | skonto
+2023-03-27T01:00:00Z | dprotaso
+2023-04-03T01:00:00Z | KauzClay
+2023-04-10T01:00:00Z | ReToCode
+2023-04-17T01:00:00Z | skonto
```

Comment:
Easter time so I will be off from 21/4-28/4

Reply:
Yeah no worries - this is best effort